cpp/ps0004/main (4).cpp

```diff
@@ -1,381 +1,32 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
-    
-  int x;
-    
-  cout<<"enter your x \n";
-  cin>>x;
+    // prompting user for number 1/1.5
+    float num;
+    cout << "enter your num \n";
+    cin >> num;
 
```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input